internal/session/repository_session_test.go

```diff
@@ -543,12 +543,14 @@ func TestRepository_updateState(t *testing.T) {
 
 func TestRepository_AuthorizeConnect(t *testing.T) {
```

Comment:
These tests should probably be moved to `repository_connection_test.go` or perhaps `service_authorize_connetion_test.go`. The test name should probably be updated as will from `TestRepository_AuthorizeConnect` to `TestService_AuthorizeConnection`